incubator/binding-mqtt/src/main/zilla/protocol.idl

```diff
@@ -102,7 +102,13 @@
             uint16 keepAlive;
             MqttProperties properties;
             string16 clientId;
-            octets payload;
+        }
+
+        struct MqttWill
+        {
+            MqttProperties properties;
+            string16 topic;
+            Binary payload;
```

Comment:
This will run into issues for large `will` messages that need to be fragmented, agree?

Reply:
As discussed, I've created a ticket for this.